stories/molecules/SelectBadge.stories.tsx

```diff
@@ -0,0 +1,218 @@
+import * as React from 'react';
```

Comment:
This should be an organism because it uses a molecule (Dropdown). Here it works because the import order is alphabetical, and Dropdown is before SelectBadge, but in other cases we had problems in the past. 
E.g. https://atomicdesign.bradfrost.com/chapter-2/ but there are other articles, this is just the first I found in case you are interested